modules/src/ics03_connection/connection.rs

```diff
@@ -7,6 +7,7 @@ use serde_derive::{Deserialize, Serialize};
 // Import proto declarations.
 use ibc_proto::connection::ConnectionEnd as ProtoConnectionEnd;
```

Comment:
I wonder if we should consider renaming the way we import proto data structures to be `as RawConnectionEnd;` instead of `as ProtoConnectionEnd;`. The same would go for `RawCounterparty` and others. The reason to do this renaming is purely aesthetic and in the interest of consistency with the `tendermint-rs` codebase (I am referring to the `RawCommitSig` struct, though I am not sure that the Raw types are analogous). WDYT?

Reply:
Good idea to keep the same naming schema. Proto was used here for the underlying implementation which you never know. Raw makes more sense. Raw, as in coming straight from the wire.

Reply:
Submitted a commit for it.